Extension/c_cpp_properties.schema.json

```diff
@@ -78,19 +83,22 @@
                         }
                     },
                     "windowsSdkVersion": {
-                        "description": "Version of the Windows SDK include path to use on Windows, e.g. '10.0.17134.0'.",
+                        "markdownDescription": "Version of the Windows SDK include path to use on Windows, e.g. `10.0.17134.0`.",
+                        "descriptionHint": "Markdown text between `` should not be translated and the punctuation should not be altered.",
                         "type": "string",
                         "pattern": "^\\d{2}\\.\\d{1}\\.\\d{5}\\.\\d{1}$|^8\\.1$"
                     },
                     "defines": {
-                        "description": "A list of preprocessor definitions for the IntelliSense engine to use while parsing files. Optionally, use = to set a value, e.g. VERSION=1.",
+                        "markdownDescription": "A list of preprocessor definitions for the IntelliSense engine to use while parsing files. Optionally, use `=` to set a value, e.g. `VERSION=1`.",
+                        "descriptionHint": "Markdown text between `` should not be translated and the punctuation should not be altered.",
                         "type": "array",
                         "items": {
                             "type": "string"
                         }
                     },
                     "intelliSenseMode": {
-                        "description": "The IntelliSense mode to use that maps to a platform and architecture variant of MSVC, gcc, or Clang. If not set or if set to ${default}, the extension will choose the default for that platform. Windows defaults to windows-msvc-x64, Linux defaults to linux-gcc-x64, and macOS defaults to macos-clang-x64. IntelliSense modes that only specify <compiler>-<architecture> variants (e.g. gcc-x64) are legacy modes and are converted automatically to the <platform>-<compiler>-<architecture> variants based on the host platform.",
+                        "markdownDescription": "The IntelliSense mode to use that maps to a platform and architecture variant of MSVC, gcc, or Clang. If not set or if set to `${default}`, the extension will choose the default for that platform. Windows defaults to `windows-msvc-x64`, Linux defaults to `linux-gcc-x64`, and macOS defaults to `macos-clang-x64`. IntelliSense modes that only specify `<compiler>-<architecture>` variants (e.g. `gcc-x64`) are legacy modes and are converted automatically to the `<platform>-<compiler>-<architecture>` variants based on the host platform.",
+                        "descriptionHint": "Uses of words 'platform', 'compiler', and 'architecture' are meant to be descriptive placeholders for literal values, so they should be translated, but without changing the surrounding `<>-<>` symbols. All other text inside `` should not translated.",
```

Comment:
Missing **be**: should not translated. -> should not **be** translated.

Reply:
Thanks: https://github.com/microsoft/vscode-cpptools/pull/8145